lisp/_prepare.el

```diff
@@ -657,14 +662,24 @@ Eask file in the workspace."
   (if eask-package-file
       (eask-error "Multiple definition of `package-file'")
     (setq eask-package-file (expand-file-name file))
-    (if (file-exists-p eask-package-file)
-        (progn
-          (with-temp-buffer
-            (insert-file-contents eask-package-file)
-            (setq eask-package-desc (ignore-errors (package-buffer-info))))
-          (unless eask-package-desc
-            (eask-warn "Failed to construct package-descriptor, lint the package-file `%s'" file)))
-      (eask-warn "Package-file seems to be missing `%s'" file))))
+    (let* ((package-file-exists (file-exists-p eask-package-file))
+           (def-point (if (eask-pkg-el) "-pkg.el file" "package-file"))
+           (target-file (cond ((eask-pkg-el) (expand-file-name (eask-pkg-el)))
+                              (package-file-exists eask-package-file))))
+      (unless package-file-exists
+        (eask-warn "Package-file seems to be missing `%s'" file))
+      (when target-file
+        (with-temp-buffer
+          (insert-file-contents target-file)
+          (setq eask-package-desc (ignore-errors
+                                    (if (eask-pkg-el)
+                                        (package--read-pkg-desc 'dir)
+                                      (package-buffer-info)))))
+        (eask-msg (concat
+                   (if eask-package-desc "✓ " "✗ ")
+                   "Try constructing the package-descriptor (%s)... "
+                   (if eask-package-desc "succeed! " "failed!"))
```

Comment:
This is a nitpick, but `succeed` is a command, meaning "I order you to succeed".

Since you are using `failed` as past tense, for success we can write `succeeded`.

Reply:
Sorry I should be more careful! Thanks! :D